.github/workflows/slack.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     steps:
     - name: Get current star count
       run: |
-        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/discuss-board-topics' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/discuss-twitter' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred discuss-board-topics! (https://github.com/layer5io/discuss-board-topics/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred discuss-twitter repo! (https://github.com/layer5io/discuss-twitter/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

Comment:
@theBeginner86 just one nitpick 😅, let's just keep it as "starred discuss-twitter!" No need to write "repo" just to maintain consistency.

Reply:
Gotcha 😁